mkdocs.yml

```diff
@@ -45,6 +45,13 @@ markdown_extensions:
 use_directory_urls: true
 plugins:
   - search
+  - exclude-search:
+      exclude:
+        - blog/*
+        - events/*
+        - staff/*
+        - trufflecon2020/*
+      exclude-unreferenced: true
```

Comment:
What does `exclude-unreferenced: true` do?

Reply:
Regarding the docs, files that are added to the docs, but are not listed in the mkdocs.yml page tree: https://github.com/chrieke/mkdocs-exclude-search/issues/17